src/index.html

```diff
@@ -13,6 +13,112 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">
+        <a href="#"></a>
```

Comment:
The <a> tags should have meaningful href attributes. Since these seem to represent stars for a rating, consider using buttons if they don't navigate to a specific URL or ensure the href attribute leads to an appropriate ID on the page.